API.md

```
@@ -0,0 +1,702 @@
+# API Documentation - AOSP ReGenesis
+
+## 📋 Table of Contents
+
+- [Architecture Overview](#architecture-overview)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)